js/components/accessibilityControls.js

```diff
@@ -3,33 +3,45 @@ var Utils = require('./utils');
 
 var AccessibilityControls = function (controller) {
   this.controller = controller;
+  this.allowed = true;
   this.state = {
     seekRate: 1,
-    lastKeyDownTime: 0
+    lastKeyDownTime: 0,
+    "keyMapsList": [
+      {"char": CONSTANTS.KEYCODES.A, "direction": "left"},
+      {"char": CONSTANTS.KEYCODES.D, "direction": "right"},
+      {"char": CONSTANTS.KEYCODES.W, "direction": "up"},
+      {"char": CONSTANTS.KEYCODES.S, "direction": "down"}
```

Comment:
Would this be simpler as a hashmap or do we expect to add more properties apart from `direction`?

```
var keyMapsList = {};
keyMapsList[CONSTANTS.KEYCODES.A] = 'left';
keyMapsList[CONSTANTS.KEYCODES.D] = 'right';
```

Reply:
\+